wireapp/wire-android-reloaded
# fix: Notifications summary fix

----
## [Bug-2019](https://wearezeta.atlassian.net/browse/AR-2019)
----

### Issues
Sometimes notifications from the app are not grouped into one big notification

### Causes (Optional)

The problem was that summaryNotification for the messages and callNotification had the same ID. 
🤦 
So when there were some notifications and call comes -> callNotification replaces summaryNotification -> notifications are not grouped anymore.

### Solutions

Created `NotificationConstants` object which stores all the notification consts and can be sure that there are no repeats.
Also added some logs to have more info in the future.

#### How to Test

- get some message notifications from different conversations
- see that notifications are grouped
- receive a call (when the app is in a background or covered by another app, so the callNotification comes)
- decline a call, or wait until the caller cancels it
- See that message notifications are still there and grouped
